fe-public/src/components/NavBar.astro

```diff
@@ -0,0 +1,86 @@
+---
+interface Props {
+    width: number;
+}
+const { width, } = Astro.props;
+---
+<div id="NavBarWidthPass" style="display: none;">{width}</div>
+
+
+<div>
+    <div id="burgerToggleContainer">
+        <button class="toggle"><img id = "burgerIcon" src="../../public/burger.png" alt="burger"></button>
+    </div>
+    <div>
+        <ul id = "navBarUL" class="navBar" style="display: none">
+            <li><a href="">Home</a></li>
+            <li><a href="../blog">Blog</a></li>
+            <hr style="border: 1px solid darkgray; margin-top:10px; margin-bottom: 10px;">
+            <li><a href="https://ie-robotics-club.notion.site/Wiki-a62b9ea0e0a14243b731c45a5f98be61" target="_blank">Notion</a></li>
+            <li><a href="https://ieconnects.ie.edu/feeds?type=club&type_id=300003041&tab=home" target="_blank">IE connects</a></li>
+            <li><a href="https://linktr.ee/ieroboticsclub" target="_blank">Socials</a></li>
+            <hr style="border: 1px solid darkgray; margin-top:10px; margin-bottom: 10px;">
+            <li><a href="https://ieconnects.ie.edu/ieroboticsai/club_signup" target="_blank"><u>Join here!</u></a></li>
```

Comment:
Maybe we could create a JSON in the directory:

```json
[ {"link": "https:/....", "name": "Notion"} ]
```

So that we can re-use the data in the footer and maybe as well in the meta #38